test/integration/options/grep.spec.js

```diff
@@ -68,40 +58,45 @@ describe('--grep', function() {
         done();
       });
     });
+  });
 
-    it('should throw an error when option used in isolation', function(done) {
-      var spawnOpts = {stdio: 'pipe'};
-      args = ['--invert'];
+  describe('when both --fgrep and --grep used together', function() {
+    it('should report an error', function(done) {
       runMocha(
-        fixture,
-        args,
+        FIXTURE,
+        ['--fgrep', 'first', '--grep', 'second'],
         function(err, res) {
           if (err) {
             return done(err);
           }
-          expect(res, 'to satisfy', {
-            code: 1,
-            output: /--invert.*--grep <regexp>/
-          });
+          expect(res, 'to have failed with output', /mutually exclusive/i);
           done();
         },
-        spawnOpts
+        'pipe'
       );
     });
   });
+});
 
-  describe('when both --fgrep and --grep used together', function() {
-    it('should conflict', function(done) {
-      // var fixture = 'uncaught.fixture.js';
-      args = ['--fgrep', 'first', '--grep', 'second'];
-
-      runMocha(fixture, args, function(err, res) {
-        if (err) {
-          return done(err);
-        }
-        expect(res, 'to have failed');
-        done();
-      });
+describe('--invert', function() {
```

Comment:
this breaks lexical ordering...

Reply:
again, if we can't automate the check or reformat automatically...

[eslint-import-order-alphabetical](https://www.npmjs.com/package/eslint-plugin-import-order-alphabetical) is probably a good place to start implementing such a thing.  it might not have a `--fix` implementation, but the check is better than nothing.

Reply:
The ordering was done by filename, each option in its own file with a single `describe`, merged just two weeks ago...

Reply:
I'll move `--invert` into its own file for consistency.  my reasoning was that a) the test was already in this file and b) the option can't be used without (f)grep

Reply:
> my reasoning was that a) the test was already in this file and b) the option can't be used without (f)grep

That's why it was previously nested...